change/@azure-msal-common-2020-11-25-19-29-14-device-code-timeout.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
```suggestion
  "type": "patch",
```

Reply:
@tnorling should the addition of a feature be classified as a `patch`?

Reply:
Since this isn't technically a full feature but an additional functionality added to polling, I would prefer a `patch` too. I do see that this case is a little gray area though.

Reply:
Cool, that's fine with me too, I don't have too strong of an opinion on either for this use case.